SPECS/heimdal/heimdal.spec

```diff
@@ -0,0 +1,869 @@
+%config(noreplace) %attr(600,root,root) %{_sysconfdir}/heimdal-kdc.conf
+%config(noreplace) %{_localstatedir}/heimdal/kdc.conf
+%config(noreplace) %attr(600,root,root) %{_sysconfdir}/heimdal-kadmind.acl
+%config(noreplace) %{_localstatedir}/heimdal/kadmind.acl
+%config(noreplace) %attr(600,root,root) %{_sysconfdir}/heimdal-slaves
+%config(noreplace) %{_localstatedir}/heimdal/slaves
+%{_libexecdir}/hprop
+%{_mandir}/man8/hprop.8*
+%{_libexecdir}/hpropd
+%{_mandir}/man8/hpropd.8*
+%{_mandir}/man8/iprop.8*
+%{_sbindir}/iprop-log
+%{_mandir}/man8/iprop-log.8*
+%{_libexecdir}/ipropd-master
+%{_mandir}/man8/ipropd-master.8*
+%{_libexecdir}/ipropd-slave
+%{_mandir}/man8/ipropd-slave.8*
+%if 0%{?_with_systemd}
+%{_libexecdir}/ipropd-slave-wrapper
+%endif
+%{_libexecdir}/%{name}-kadmind
+%{_mandir}/man8/%{name}-kadmind.8*
+%{_libexecdir}/kdc
+%{_mandir}/man8/kdc.8*
+%{_libexecdir}/kpasswdd
+%{_mandir}/man8/kpasswdd.8*
+%{_sbindir}/kstash
+%{_mandir}/man8/kstash.8*
+
+%files workstation
+%{_bindir}/afslog
+%{_mandir}/man1/afslog.1*
+%{_bindir}/bsearch
+%{_mandir}/man1/bsearch.1*
+%{_bindir}/%{name}-pagsh
+%{bindir}/bin/pagsh
+%{_mandir}/man1/%{name}-pagsh.1*
+%{_bindir}/gsstool
+%{_bindir}/heimtools
+%{_bindir}/hxtool
+%{_bindir}/idn-lookup
+%{_bindir}/%{name}-kdestroy
+%{bindir}/bin/kdestroy
+%{_mandir}/man1/%{name}-kdestroy.1*
+%{_bindir}/kf
+%{_mandir}/man1/kf.1*
+%{_bindir}/kgetcred
+%{_mandir}/man1/kgetcred.1*
+%{_libexecdir}/kimpersonate
+%{_mandir}/man8/kimpersonate.8*
+%{_bindir}/%{name}-kinit
+%{bindir}/bin/kinit
+%{_bindir}/kauth
+%{_mandir}/man1/%{name}-kinit.1*
+%{_bindir}/%{name}-klist
+%{bindir}/bin/klist
+%{_mandir}/man1/%{name}-klist.1*
+%{_bindir}/%{name}-kpasswd
+%{bindir}/bin/kpasswd
+%{_mandir}/man1/%{name}-kpasswd.1*
+%{_bindir}/heimdal-kswitch
+%{bindir}/bin/kswitch
+%{_mandir}/man1/heimdal-kswitch.1*
+%{_bindir}/otp
+%{_mandir}/man1/otp.1*
+%{_bindir}/otpprint
+%{_mandir}/man1/otpprint.1*
+%{_bindir}/%{name}-kadmin
+%{bindir}/bin/kadmin
+%{_mandir}/man1/%{name}-kadmin.1*
+%{_libexecdir}/kcm
+%{_mandir}/man8/kcm.8*
+%{_libexecdir}/kfd
+%{_mandir}/man8/kfd.8*
+%{_bindir}/%{name}-ktutil
+%{bindir}/bin/ktutil
+%{_mandir}/man1/%{name}-ktutil.1*
+# NOTICE: no support for X11
+#%%{_libexecdir}/kxd
+#%%{_mandir}/man8/kxd.8*
+#%%{_mandir}/cat8/kxd.8*
+%attr(04550,root,root) %{_bindir}/%{name}-su
+%{bindir}/bin/su
+%{_mandir}/man1/%{name}-su.1*
+
+%files devel
+%dir %{_libexecdir}/%{name}
+%{_bindir}/%{name}-krb5-config
+%{bindir}/bin/krb5-config
+%{_mandir}/man1/%{name}-krb5-config.1*
+%{_includedir}/*
+%{libdir}/lib*.a
+%{libdir}/windc.a
+%{_mandir}/man3/*
+%{_mandir}/man7/*
+%{_libexecdir}/%{name}/asn1_compile
+%{_libexecdir}/%{name}/asn1_print
+%{_libexecdir}/%{name}/slc
+
+%files path
+%{_sysconfdir}/profile.d/%{name}.sh
+%{_sysconfdir}/profile.d/%{name}.csh
+
+%changelog
+* Wed Oct 21 2020 Henry Beberman <henry.beberman@microsoft.com> - 7.7.0-5
+- Initial CBL-Mariner import from Fedora 32 (license: MIT).
```

Comment:
Please check the license and add the phrase `License verified`. This way we're sure that someone has already looked at the licenses on our end.